### PR TITLE
remove feature flag high contrast mode

### PR DESCRIPTION
**Pull request by lisli1**
#### Pull request checklist

- [ ] Addresses an existing issue: Fixes #0000 **N/A**
- [x] Added relevant unit test for your changes. (`npm run test`)
- [x] Verified code coverage for the changes made. Check coverage report at: `<rootDir>/test-results/unit/coverage`
- [x] Ran precheckin (`npm run precheckin`)
- [ ] Added screenshots/GIFs for UI changes. **N/A**

#### Description of changes

Removed highContrastMode from FeatureFlags and all references to FeatureFlags.highContrastMode

#### Notes for reviewers

(put down any questions or special points for reviewers)

### CODE REVIEW

File: src/tests/unit/tests/DetailsView/components/settings-panel.test.tsx [hunk 1]
```diff
@@ -107,11 +108,11 @@ describe('SettingsPanelTest', () => {
                 userConfigMessageCreator: userConfigMessageCreatorMock.object,
             },
             userConfigStoreState: userConfigStoreData,
-            featureFlagData: { [FeatureFlags.highContrastMode]: false },
+            featureFlagData: { 'test-flag': false },
```
Comment (dbjorge) [inline src/tests/unit/tests/DetailsView/components/settings-panel.test.tsx]: >featureFlagData: { 'test-flag': false }, [](start = 12, length = 40)

This shouldn't have any effect on the tests that are passing `'test-flag'`, right? can you instead just omit it?

Comment (lisli1) [reply src/tests/unit/tests/DetailsView/components/settings-panel.test.tsx]: The problem is that SettingsPanelProps expects a featureFlagData element. I have tried removing it completely but it gives me an error, so I decided to replace the feature flag string with 'test-flag'

Comment (smoralesd) [reply src/tests/unit/tests/DetailsView/components/settings-panel.test.tsx]: @dbjorge the reason I asked @lisli1 to do this is a bit more profound:
1. As part of this PR, `SettingsPanel` component is no longer using `FeatureFlags.highContrastMode` directly.
2. `SettingsPanelProps` still have a `featureFlagData` property, which we are passing through to the `FlaggedComponent` to render the bug filling setting.
3. As far as I know (please correct me if I'm wrong), from the snapshot test point of view, all empty objects are made equal.

Point 3. is the main reason for having this "fake" feature flag key under `featureFlagData`. In order to properly assert the `featureFlagData` is indeed being passed through to the `FlaggedComponent` we should not use an empty object. Instead, we should a test object, with test data, which will make it easier for the future dev to check that this condition is actually being met. Does this make sense?

File: src/tests/unit/tests/DetailsView/components/settings-panel.test.tsx [hunk 2]
```diff
@@ -107,11 +108,11 @@ describe('SettingsPanelTest', () => {
...
         };
 
-        const testSubject = new TestableSettingsPanel(testProps);
-        expect(testSubject.render()).toMatchSnapshot();
+        const wrapped = shallow(<SettingsPanel {...testProps} />);
+        expect(wrapped.getElement()).toMatchSnapshot();
```
Comment (Shobhit1) [inline src/tests/unit/tests/DetailsView/components/settings-panel.test.tsx]: Love this. `shallow` is a much better way to test with snapshot!

**Pull request merged**